frontend/components/System/SelectedDlnaProfile.vue

```diff
@@ -0,0 +1,924 @@
+<template>
+  <v-card class="selected-dlna-profile">
+    <v-row v-if="isDialog" class="ma-0 justify-space-between align-center">
+      <v-card-title>{{ selectedProfile.Name }}</v-card-title>
+      <v-btn icon class="mr-2" @click="$emit('close-dialog')">
+        <v-icon>mdi-close</v-icon>
+      </v-btn>
+    </v-row>
+    <v-card-title v-else>{{ currentProfile.Name }}</v-card-title>
+    <v-tabs v-model="tab" centered dark icons-and-text>
+      <v-tabs-slider />
+      <v-tab v-for="i in 9" :key="i" :href="'#tab-' + i">
+        {{
+          $t(
+            'settings.dlna.profile.' +
+              categories[i - 1].toLocaleString() +
+              '.title'
+          )
+        }}
+      </v-tab>
+    </v-tabs>
+    <v-tabs-items v-model="tab">
+      <v-tab-item value="tab-1">
+        <v-list>
+          <!-- General -->
```

Comment:
As you say, this file is massive. However, why it can't be ``v-for`` _looped_? You can loop over ``currentProfile`` with Object.entries like ``(key, value) in Object.entries(currentProfile)`` and bind the value to ``v-model``.

Reply:
I'm currently working on this, however, the way you suggested is not feasible as we would modify data of the loop within it. (v-modeling `value` results in this eslint warning being thrownhttps://eslint.vuejs.org/rules/valid-v-model.html)

Reply:
Try ignoring the rule, might be a false positive but work perfectly during runtime.